components/partials/home/HomeWelcome.vue

```diff
@@ -121,6 +121,13 @@ export default {
   components: {
     MeteorIcon,
     GithubIcon
+  },
+  computed: {
+    videoUrl() {
+      return this.$i18n.locale === 'id'
```

Comment:
nit: Ideally leave a comment here linking to this PR (or alternatively describing why the video URL is different for the indonesian language.

Reply:
I did it so that the video display appeared in Indonesia. because the vimeo.com site is prohibited or blocked in Indonesia. So it's no wonder I do this so that the video continues to appear on nuxtjs.org by using the youtube service for the video

Reply:
Yes, I'm aware of that. But I think having at least a reference to that PR (or a short explanation) *in the code* would help future readers to understand why the code has been changed like that :)

Reply:
I've fixed it, you mean like that

Reply:
Yes, perfect 👍🏻